app/javascript/controllers/flash_message_controller.js

```diff
@@ -0,0 +1,8 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  connect() {
+    var toast = new bootstrap.Toast(this.element, { delay: 3500 })
```

Comment:
The default delay of 5s seemed too long to me so I adjusted it to 3.5s. Let me know if it's ok

Reply:
Sounds reasonable thank you!